### PR TITLE
Fix grammar in README

### PR DESCRIPTION
**Pull request by yonasstephen**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-2. Stub server that serving on `http://localhost:4771`. It's main job is to store all the stub mapping. We can add a new stub or list existing stub using http request.
+1. GRPC server that serves on `tcp://localhost:4770`. Its main job is to serve incoming rpc call from client and then parse the input so that it can be posted to Stub service to find the perfect stub match.
+2. Stub server that serves on `http://localhost:4771`. Its main job is to store all the stub mapping. We can add a new stub or list existing stub using http request.
 
 Matched stub will be returned to GRPC service then further parse it to response the rpc call.
 
 ## Quick Usage
-First, prepare your `.proto` file. or you can use `hello.proto` in `example/pb/` folder. Suppose you put it in `/mypath/hello.proto`. We gonna use Docker image for easier example test.
+First, prepare your `.proto` file. oOr you can use `hello.proto` in `example/pb/` folder. Suppose you put it in `/mypath/hello.proto`. We are gonna use Docker image for easier example test.
```
Comment (damienhollis) [inline Readme.md:14]: Typo `oOr` should be `Or`

**Review by damienhollis — CHANGES_REQUESTED** [review]
One minor typo

File: Readme.md (hunk 1 above)
Comment (yonasstephen) [reply Readme.md]: Fixed.

File: Readme.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
 # GripMock
-GripMock is a **mock server** for **GRPC** services. It's using `.proto` file to generate implementation of gRPC service for you.
-If you already familiar with [Apiary](https://apiary.io) or [WireMock](http://wiremock.org) for mocking API service and looking for similiar thing for GRPC then this is the perfect fit for that.
+GripMock is a **mock server** for **GRPC** services. It's using a `.proto` file to generate implementation of gRPC service for you.
```
Comment (jekiapp) [inline Readme.md:2]: it can be more than one

**Review by jekiapp — APPROVED** [review]

**Pull request merged**